### ISSUE TITLE
Enable resizing of text area when responding

### ISSUE THREAD

**Issue by jsorva** [apluslms/mooc-jutut#77]
Before the recent UI update, it was possible freely to resize the teacher’s response box both vertically and horizontally by dragging the corner of the relevant text are (highlighted below). This saw quite a bit of use, at least by me.
![image](https://github.com/apluslms/mooc-jutut/assets/8523129/edda269c-1a5c-4b08-93fb-67d5559ac77b)

After the update, the resizing only works vertically, which is unfortunate. 

(This is not urgent, but hopefully can be fixed at some point.)

**Comment by etanttila** [comment]
This should be very easy to fix, one just has to change `resize: vertical` to `resize: both` in [feedback.css](https://github.com/apluslms/mooc-jutut/blob/407a2434f02d5db196e2251c5d966744f1476298/feedback/static/feedback.css#L476). I removed the possibility to resize the textbox horizontally, since I didn't realize that it was a desired funtionality (since it does make it look kind of weird that the textbox comes out of the container width-wise.

**Comment by Mankro** [comment]
@ihalaij1 note Ella's comment. It might be helpful to you.
